docs/tutorial/part-one/index.md

```diff
@@ -21,6 +21,8 @@ gatsby new [SITE_DIRECTORY_NAME] [URL_OF_STARTER_GITHUB_REPO]
 
 If you omit a URL from the end, Gatsby will automatically generate a site for you based on the [**default starter**](https://github.com/gatsbyjs/gatsby-starter-default). For this section of the tutorial, stick with the “Hello World” site you already created in tutorial part zero.
 
+> 💡 You can also create a new site from a local starter by running: `gatsby new [SITE_DIRECTORY_NAME] [PATH/TO/LOCAL/STARTER]`
+
```

Comment:
Can we make change like this in separate PR and have only actual fix in this one?

We try not to overload tutorial with every possible way you can create new gatsby project, so I don't think this is best place for it. I would put this in https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/gatsby-cli.md

Reply:
I will do it. Thanks for the quick reply

Reply:
I've removed the commit with the docs, leaving only the code fix on this PR as suggested.

I've also looked into your suggestion of opening another PR with the doc change on the `gatsby-cli.md` file instead of the tutorial. But when reading the content of the `gatsby-cli.md` this is what is currently there for the `new` command:

```
### `new`

`gatsby new gatsby-site`

See the [Gatsby starters docs](/docs/starters/)
for a comprehensive list of starters to get started with Gatsby.
```

So I will be submiting another PR, but to the https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/starters.md instead.